vscode: Add a walkthrough to help getting started with Slint (#3292)
slint-ui/slint: editors/vscode/static/walkthroughs/welcome/create_slint_file.md

<!-- Copyright © SixtyFPS GmbH <info@slint.dev> ; SPDX-License-Identifier: MIT -->

The HelloWorld for Slint looks like this:

```slint
import { Button, VerticalBox } from "std-widgets.slint";

export component Demo {
    VerticalBox {
        alignment: start;
        Text {
            text: "Hello World!";

>             font-size: 24px;

Reviewer: ```suggestion
            font-size: 2rem;
```

            horizontal-alignment: center;
        }
        HorizontalLayout {
            alignment: center;

>             Button { text: "OK!"; }

Reviewer: No shouting exclamation marks on UI controls IMO.

```suggestion
            Button { text: "OK"; }
```

        }
    }
}
```

> You can copy and paste that into your new `.slint` file to get you started.

Reviewer: ```suggestion
To get started, copy and paste this into your new `.slint` file.
```

> _Make sure to save the file with .slint extension for VSCode to accept that it is indeed a Slint file_

Reviewer: ```suggestion
_Make sure to save the file with .slint extension for VSCode to accept that it is indeed a Slint file._
```
